src/platforms/common/syscalls.c

```diff
@@ -186,3 +186,9 @@ __attribute__((used)) void *_sbrk(const ptrdiff_t alloc_size)
 	heap_current += alloc_size;
 	return result;
 }
+
+/* ARM EABI Personality function 0 for newlib-4.3.0 */
+__attribute__((weak)) void __aeabi_unwind_cpp_pr0(void)
+{
+	abort();
```

Comment:
Yes, libgcc is exactly where I looked for arguments, and even in my first iteration of this I defined them and consumed as unused. But linker seems to not care about it.
Are you sure we don't want to halt firmware on unwind events etc.? I can drop the abort preemptively, but I don't know when this function ever gets to be called.

Reply:
The problem we foresee is that in calling `abort()`, it'll re-trigger the unwinder, making this code trigger *again* and spiralling into uncontrolled stack usage and a crash, which is not especially useful seeing the only usage of `setjmp()`/`longjmp()` in the code base is for error handling which we want to have progress normally regardless so that normal error handling can take place and do its thing.

Reply:
Fair. Dropped `abort()`. Added pr1, pr2.